### DISCUSSION TITLE
Is this project abandoned?

### DISCUSSION THREAD

**Discussion by vivgui** [uicrooks/shopify-theme-lab#175] [Q&A]
The last commit was on Dec 2021. PRs are not getting reviewed. The Shopify CLI was updated and broke one of the NPM scripts and hasn't been updated.

If the owners can't or don't want to maintain it anymore, should someone fork it?

**Comment by james0r** [comment]
Saw that last commit and was wondering the same thing today.

**Comment by ghost** [comment]
Same thing here. Wondering how we should approach this setup with CLI 3+.
